test.sh

```diff
@@ -140,6 +140,13 @@ generateSolLog(){
         bash generate-sol-log.sh' > ${WORKSPACE}/sol.log &
 }
 
+setupVirtualEnv(){
+  cd ${WORKSPACE}/RackHD/test
+  rm -rf .venv/on-build-config
+  ./mkenv.sh on-build-config
+  source myenv_on-build-config
```

Comment:
Just double-checking.  There is no need to cd back out of RackHD/test at the end, is there?